packages/fxa-auth-server/config/dev.json

```diff
@@ -427,7 +427,7 @@
     "secondInterval": "2s"
   },
   "mjml": {
-    "enabledEmailAddress": "@testuser.com$",
+    "enabledEmailAddress": "@testuser.com$|@b.com",
```

Comment:
I'm guessing the `|@b.com` this is just for testing and will be removed? I don't mind leaving `@testuser.com`

Reply:
Yeah I think we're using `a@b.com` address with our tests